Preserve window size and position, also get and set

michael-dewar opened the pull request:
This PR will preserve the size and position of the window when you open the xlsx in Excel.  There are also functions to set and get directly.

michael-dewar committed:
preserve window size and position on loadWorkbook

michael-dewar committed:
preserve window size and position on saveWorkbook

michael-dewar committed:
get and set window size

michael-dewar committed:
fix document() errors

michael-dewar committed:
fix hyperlink in documentation

michael-dewar committed:
export function

JanMarvin commented:
Hi @michael-dewar , looking good. Just minor lintr issues and I'd like to see a test for this as well.

michael-dewar committed:
add tests, also linter fixes

michael-dewar commented:
I fixed the linter issues for the code I changed.  I mostly avoided changing the other sections because I wasn't actually sure if  the & vs && change made a difference compared to what was intended.

I also added tests.

JanMarvin commented:
Thanks, the other changes should be in the `development` branch

ycphs committed:
Merge branch 'development' into pr/383
